pkg/etcdv3/watch.go

```diff
@@ -209,9 +213,6 @@ func (e *etcdV3) Watch(stream etcdserverpb.Watch_WatchServer) error {
 		ctx:      ctx,
 	}
 	e.addWatchStream(ws)
-	e.logger.Debugw("add new watcher",
```

Comment:
Why add this change?

Reply:
This log is about `addWatchStream`, so move it inside that function